data/mods/Fuji_Structures/modinfo.json

```diff
@@ -1,13 +1,12 @@
-{
-  "type": "MOD_INFO",
-  "mod-type": "SUPPLEMENTAL",
-  "ident": "FujiStruct",
-  "name": "Fuji's More Buildings",
-  "description": "Adds more buildings and more variations to existing buildings. (Requires More Locations)",
-  "category": "buildings",
-  "authors": "Fuji",
-  "dependencies": [
-    "dda",
-    "more_locations"
-  ]
-}
+[
+  {
+    "type": "MOD_INFO",
+    "mod-type": "SUPPLEMENTAL",
```

Comment:
`"mod-type"` node is redundant and not supported by current `master` branch. All mods are considered supplemental until they have `"core": true` node.

Reply:
Removed that entry.